templates/include/telemetry/mainPayload.1.schema.json

```diff
@@ -55,7 +53,7 @@
       "type": "object",
       "additionalProperties": {
         "additionalProperties": {
-          @TELEMETRY_PROCESSDATA_1_JSON@
+          @TELEMETRY_HISTOGRAM_1_JSON@
```

Comment:
@trink I noticed this while looking at the nested `additionalProperties` too - looks like this should be including the histogram definition rather than the "processData" one as it was previously. Does that sound right?